pages/common/youtube-dl.md

```diff
@@ -33,4 +33,4 @@
 
 - Download a playlist and extract mp3 from it:
 
-`youtube-dl -i --extract-audio --audio-format mp3 -o "%(title)s.%(ext)s" '{{url_to_playlist}}'`
+`youtube-dl -f "bestaudio" -cwix --audio-format mp3 -o "%(title)s.%(ext)s" {{url_to_playlist}}`
```

Comment:
It's a bit difficult to know what `-cwix` means here in this context. Generally we prefer the long-forms of arguments in tldr-pages.

Reply:
```suggestion
`youtube-dl -f "bestaudio" --continue --no-overwrites --ignore-errors --extract-audio --audio-format mp3 -o "%(title)s.%(ext)s" {{url_to_playlist}}`
```